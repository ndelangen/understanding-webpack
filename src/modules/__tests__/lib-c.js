import * as c from '../lib-c';

describe('sanity', () => {
  it('should exist', () => {
    expect(c).toBeDefined();
  });
  it('should contain the right methods', () => {
    expect(Object.keys(c)).toMatchSnapshot();
  });
});
