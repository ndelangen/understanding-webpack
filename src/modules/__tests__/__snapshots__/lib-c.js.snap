// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`sanity should contain the right methods 1`] = `
Array [
  "myFunc",
  "MY_CONST",
  "myArrowFunc",
  "MyClass",
  "foo",
  "bar",
]
`;
